.github/triage_replies.md

```diff
@@ -3,37 +3,81 @@
 - Hello, we think your question is answered in our FAQ. Does this: https://www.ansible.com/products/awx-project/faq cover your question?
 - You can find the latest documentation here: https://docs.ansible.com/automation-controller/latest/html/userguide/index.html
 
-## Visit our mailing list
+
+
+## PRs
+
+### Visit our mailing list
 - Hello, your question seems like a good one to ask on our mailing list at https://groups.google.com/g/awx-project. You can also join #ansible-awx on https://libera.chat/ and ask your question there.
 
-## Create an issue
-- Hello, thanks for reaching out on list. We think this merits an issue on our Github, https://github.com/ansible/awx/issues. If you could open an issue up on Github it will get tagged and integrated into our planning and workflow. All future work will be tracked there.
+### Denied Submission
 
-## Create a Pull Request
-- Hello, we think your idea is good, please consider contributing a PR for this, following our contributing guidelines: https://github.com/ansible/awx/blob/devel/CONTRIBUTING.md
+- Hi!\
+\
+Thanks very much for your submission to AWX. It means a lot to us that you have taken time to contribute.\
+\
+At this time we do not want to merge this PR. Our reasons for this are:\
+\
+(A) INSERT ITEM HERE\
+\
+Please know that we are always up for discussion but this project is very active. Because of this, we're unlikely to see comments made on closed PRs and we lock them after some time. If you or anyone else has any further questions, please let us know by using any of the communication methods listed in the page below:\
+\
+https://github.com/ansible/awx/#get-involved\
+\
+In the future, sometimes starting a discussion on the development list prior to implementing a feature can make getting things included a little easier, but it is not always necessary.\
+\
+Thank you once again for this and your interest in AWX!
+
+
+### No Progress
+- Hi!\
+\
+Thank you very much for your submission to AWX. It means a lot to us that you have taken time to contribute.\
+\
+On this PR changes were requested but it has been some time since then. We think this PR has merit but without the requested changes we are unable to merge it. At this time we are closing you PR. If you get time to address the changes you are welcome to open another PR or we can reopen this PR upon request if you contact us by using any of the communication methods listed in the page below:\
```

Comment:
I don't think we have anything like that but we should. It would be nice if we could add an "inactive" label which would reply with this and then close in 1mo if no additional activity happens on the PR.

Reply:
the ansible-core team has a bot that does this. I don't know any of the details but I think this is the code - https://github.com/ansible/ansibullbot.  It does a LOT of handy things wrt pr and issue mgmt.

Reply:
I think we used to have an 'awxbot' that was similar to ansibullbot but I'm not sure why we stopped using it.